### ISSUE TITLE
st_minimum_bounding_circle

### ISSUE THREAD

**Issue by Nowosad** [r-spatialecology/vectormetrics#18]
@TomekMatuszek -- can `st_minimum_bounding_circle()` (https://github.com/r-spatial/sf/issues/2473#event-15442430858) be useful somewhere in the package?

**Comment by TomekMatuszek** [comment]
Yes, it would be very useful. Actually, I wrote simmilar function myself - `get_circum_diam`. I will check if there is an improvement in performance (probably yes) next week.
